modules/data.atmosphere/R/download.NARR.R

```diff
@@ -7,6 +7,7 @@
 ##' @param verbose Turn on verbose output? Default=FALSE
 ##' @param method Method of file retrieval. Can set this using the options(download.ftp.method=[method]) in your Rprofile.
 ##' example options(download.ftp.method="ncftpget")
+##' @importFrom magrittr %>%
```

Comment:
For imports like this that are likely to get used in multiple places, I'd prefer we make a habit of putting them in a place that clearly applies to the whole package, e.g. in  PEcAn.DB we put the importFrom in the [package documentation block](https://github.com/PecanProject/pecan/blob/develop/base/db/R/zzz.R#L8). 

That said, maybe it makes sense to import it here first and then move it when it's actually being used multiple places in the package.